components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/debugger/debugger.js

```diff
@@ -219,7 +220,22 @@ define(['jquery', 'backbone', 'log','lodash','ace/range','render_json'], functio
             debuggerModel.find(".fw-resume").click(function(e) {
                 e.preventDefault();
                 self._debugger.play();
+                //console.log("debugger resume click");
             });
+            // this._resumeBtn.on('click', function (e) {
```

Comment:
Please remove the commented lines

Reply:
Fixed by 1a6fb09c8eacdc06dcd93c202fa8abdcebef3c5e